src/lavinmq/config.cr

```diff
@@ -210,30 +219,32 @@ module LavinMQ
-        when "log_level"                 then @log_level = ::Log::Severity.parse(v)
-        when "log_file"                  then @log_file = v
-        when "stats_interval"            then @stats_interval = v.to_i32
-        when "stats_log_size"            then @stats_log_size = v.to_i32
-        when "segment_size"              then @segment_size = v.to_i32
-        when "set_timestamp"             then @set_timestamp = true?(v)
-        when "socket_buffer_size"        then @socket_buffer_size = v.to_i32
-        when "tcp_nodelay"               then @tcp_nodelay = true?(v)
-        when "tcp_keepalive"             then @tcp_keepalive = tcp_keepalive?(v)
-        when "tcp_recv_buffer_size"      then @tcp_recv_buffer_size = v.to_i32?
-        when "tcp_send_buffer_size"      then @tcp_send_buffer_size = v.to_i32?
-        when "tls_cert"                  then @tls_cert_path = v
-        when "tls_key"                   then @tls_key_path = v
-        when "tls_ciphers"               then @tls_ciphers = v
-        when "tls_min_version"           then @tls_min_version = v
-        when "guest_only_loopback"       then @guest_only_loopback = true?(v)
-        when "log_exchange"              then @log_exchange = true?(v)
-        when "free_disk_min"             then @free_disk_min = v.to_i64
-        when "free_disk_warn"            then @free_disk_warn = v.to_i64
-        when "max_deleted_definitions"   then @max_deleted_definitions = v.to_i
-        when "consumer_timeout"          then @consumer_timeout = v.to_u64
-        when "default_consumer_prefetch" then @default_consumer_prefetch = v.to_u16
+        when "data_dir"                   then @data_dir = v
+        when "data_dir_lock"              then @data_dir_lock = true?(v)
+        when "log_level"                  then @log_level = ::Log::Severity.parse(v)
+        when "log_file"                   then @log_file = v
+        when "stats_interval"             then @stats_interval = v.to_i32
+        when "stats_log_size"             then @stats_log_size = v.to_i32
+        when "segment_size"               then @segment_size = v.to_i32
+        when "set_timestamp"              then @set_timestamp = true?(v)
+        when "socket_buffer_size"         then @socket_buffer_size = v.to_i32
+        when "tcp_nodelay"                then @tcp_nodelay = true?(v)
+        when "tcp_keepalive"              then @tcp_keepalive = tcp_keepalive?(v)
+        when "tcp_recv_buffer_size"       then @tcp_recv_buffer_size = v.to_i32?
+        when "tcp_send_buffer_size"       then @tcp_send_buffer_size = v.to_i32?
+        when "tls_cert"                   then @tls_cert_path = v
+        when "tls_key"                    then @tls_key_path = v
+        when "tls_ciphers"                then @tls_ciphers = v
+        when "tls_min_version"            then @tls_min_version = v
+        when "default_user_only_loopback" then @default_user_only_loopback = true?(v)
```

Comment:
What about still supporting `guest_only_loopback` but log a warning if it's used?